platform/os_windows.go

```diff
@@ -13,13 +13,17 @@ import (
 	"syscall"
```

Comment:
We are changing only the functions used by CNI right ? I see `log` and `zaplogger` both in this file

Reply:
yes that's correct, we only change functions called by CNI, other packages are still using legacy log